### PR TITLE
more cleanup and removing mentions of the "retrieve* button 

### PR DESCRIPTION
**Pull request by KZzizzle**
#8  What do these changes do?

A few more changes for accuracy and removing mentions of the "retrieve button" which doesn't exist anymore

## Checklist

- [x] I think the documentation is well written and up to date
- [ ] Do NOTE have very large media files (e.g. photos, videos, etc) - nope, I compressed them!
- [x] My spell-checker passes
- [ ] If you write a new section, i take ownership adding my user in .github/CODEOWNERS

### CODE REVIEW

File: docs/isan_studies___tutorials/uc_davies.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ To explore the service and the model, please proceed as follows:
 
     ![CC_inputfiles](https://user-images.githubusercontent.com/32800795/61584726-1d111100-ab4d-11e9-9a91-93c1da82b05c.JPG ':size=500%')
 
-3. Next, click “Retrieve” at the bottom of the worbench and then “Run.” You will then be able to view results of the simulation in any of the viewer services.
+3. Next, click “Run” at the bottom of the worbench. You will then be able to view results of the simulation in any of the viewer services.
```
Comment (pcrespov) [inline docs/isan_studies___tutorials/uc_davies.md:17]: typo: wor**K**bench

Perhaps a spell-checker in vscode can help detecting these typos on the fly
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

**Review by pcrespov — CHANGES_REQUESTED** [review]

File: docs/isan_studies___tutorials/uc_davies.md (hunk 1 above)
Comment (pcrespov) [reply docs/isan_studies___tutorials/uc_davies.md:17]: could also add an auto-check either in pre-commit or as a git action for all manuals

File: _sidebar.md [hunk 2]
```diff
@@ -20,7 +20,7 @@
   * [Connecting Services](/docs/setting_up___running_a_study/connecting_services.md)
   * [Jupyter Notebooks](/docs/setting_up___running_a_study/jupyter_notebooks.md)
   * [JupyterLab](/docs/setting_up___running_a_study/JupyterLabs.md)
-  * [Running a Simulation](/docs/setting_up___running_a_study/running_a_model/simulation.md)
+  * [Running a Pipeline](/docs/setting_up___running_a_study/run_simulation.md)
   * [Visualization](/docs/setting_up___running_a_study/postprocessing___analysis/postpro_viewer_types.md)
```
[the remark this answers is not in the document]
Comment (KZzizzle) [reply _sidebar.md:24]: ask Nik =) I was too lazy to change the folder name and all the paths.

File: docs/isan_studies___tutorials/2d_plot.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ My Data -> simcor.s3 -> PlotAdventure -> File Picker
 
 ![plot2d_tree](https://user-images.githubusercontent.com/32800795/61495276-cbd01880-a9b8-11e9-8b3b-5bc44584df8d.JPG ':size=550%')
 
-Here you should see your uploaded file RNAdat.csv. Double click to select it and it should now be visible on the file name field of the File Picker service. Click “Retrieve at the bottom of the workbench to retrieve the file.
+Here you should see your uploaded file RNAdat.csv. Double click to select it and it should now be visible on the file name field of the File Picker service.
```
[the remark this answers is not in the document]
Comment (KZzizzle) [reply docs/isan_studies___tutorials/2d_plot.md:25]: changed

File: docs/isan_studies___tutorials/uc_davies.md (hunk 1 above)
Comment (KZzizzle) [reply docs/isan_studies___tutorials/uc_davies.md:17]: my concern with the git action is that we have a lot of "wrong" words like "osparc", "neuro-", filenames, code snippets etc. I installed the spell-checker you suggested

**Review by pcrespov — APPROVED** [review]
beautiful! :-)

**Pull request merged**